src/utils/init/config-github.js

```diff
@@ -142,88 +142,42 @@ async function configGithub(ctx, site, repo) {
     }
   }
 
+  ctx.log()
+  ctx.log(`Creating Netlify Github Notification Hooks...`)
+
   // TODO: Generalize this so users can reset these automatically.
   // Quick and dirty implementation
   const ntlHooks = await api.listHooksBySiteId({ siteId: site.id })
 
-  const createdHook = ntlHooks.find((h) => h.type === 'github_commit_status' && h.event === 'deploy_created')
-  const failedHook = ntlHooks.find((h) => h.type === 'github_commit_status' && h.event === 'deploy_failed')
-  const buildingHook = ntlHooks.find((h) => h.type === 'github_commit_status' && h.event === 'deploy_building')
+  await upsertHook({ ntlHooks, event: 'deploy_created', api, site, ghtoken })
+  await upsertHook({ ntlHooks, event: 'deploy_failed', api, site, ghtoken })
+  await upsertHook({ ntlHooks, event: 'deploy_building', api, site, ghtoken })
```

Comment:
Are we able to run those 3 in parallel? I kept it serial just in case.

Reply:
This is a great cleanup 🧹  Looks like we should be able to parallelise this. We can do it in another PR as it will require some manual testing